published/33-tormath1.md

```diff
@@ -0,0 +1,7 @@
+Flatcar Rawkode Academy Ep. 1
+
+### Content
+
+Grab your favorite drink, set yourself "in a meeting" and watch this last episode from @RawkodeAcademy about Flatcar to learn or confirm your knowledge. Thanks @rawkode for the invit and for maintaining this collection of Cloud Native interviews. 
```

Comment:
```suggestion
Grab your favorite drink, set yourself "in a meeting" and watch this last episode from @RawkodeAcademy about Flatcar to learn or confirm your knowledge. Thanks @rawkode for the invite and for maintaining this collection of Cloud Native interviews. 
```
or maybe "invitation"

Should this also include the link?

Reply:
Done thanks. Yeah, the link will be embedded (see: https://github.com/flatcar/socials/issues/33#issuecomment-3484800117)
I just don't add it in the initial post to avoid make the Twitter count > 280 chars (URL is shrinked when added to a Twitter post and we don't really know the size in advance).